File: backend/src/main/java/objects/serializable/News.java
package objects.serializable;

public class News {
    private String title;
    private String context;
    private String tag;

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getTag() {
        return tag;
    }

    public void setTag(String tag) {
        this.tag = tag;
    }

    public String getContext() {
        return context;
    }

    public void setContext(String context) {
        this.context = context;
    }
}

File: frontend/lib/objects/News.dart
import 'dart:collection';

import 'package:flutter/cupertino.dart';
import 'package:font_awesome_flutter/font_awesome_flutter.dart';
import 'package:frontend/objects/date-hour.dart';
import 'Teacher.dart';

class News{
  late String title;
  late String context;
  late String tag;

  News({required this.title, required this.context, required this.tag});

  factory News.fromJson(Map<String, dynamic> json) {
     return News(
         title: json['title'],
         context: json['context'],
         tag: json['tag']
     );
  }

}
